src/views/FindOrder.vue

```diff
@@ -104,61 +104,80 @@
+                      {{ item.virtualProductName }}
+                      <p> {{ $t("Color") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/COLOR/') }} </p>
+                      <p> {{ $t("Size") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/SIZE/') }} </p>
                     </ion-label>
-                    <ion-badge color="primary" slot="end">Approved</ion-badge>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("Promise date") }} </ion-label>
-                    <p slot="end">14 Jan 2021</p>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("PO arrival date") }} </ion-label>
-                    <p slot="end">14 Jan 2021</p>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("Last brokered") }} </ion-label>
-                    <p slot="end">California Warehouse</p>
+                    <ion-badge color="primary" slot="end"> {{ item.orderItemStatusId }} </ion-badge>
                   </ion-item>
+                  <!-- TODO: Need to handle this property -->
+                  <div v-if="item.preOrderStatus || item.backOrderStatus || item.unFillable">
+                    <ion-item>
+                      <ion-label> {{ $t("Promise date") }} </ion-label>
+                      <p slot="end"> {{ item.promisedDatetime ? $filters.formatUtcDate(item.promisedDatetime, 'YYYY-MM-DDTHH:mm:ssZ') : '-'  }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("PO arrival date") }} </ion-label>
+                      <!-- TODO: Need to handle this property -->
+                      <p slot="end"> {{ item.promiseOrderArrivalDate ? item.promiseOrderArrivalDate : '-' }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("Last brokered") }} </ion-label>
+                      <!-- TODO: Need to handle this property -->
+                      <p slot="end"> {{ item.lastBrokered ? item.lastBrokered : '-' }} </p>
+                    </ion-item>
+                  </div>
+                  <div v-else>
+                    <ion-item>
+                      <ion-label> {{ $t("Shipping method") }} </ion-label>
+                      <p slot="end"> {{ item.shipmentMethodTypeId }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("Shipping from") }} </ion-label>
+                      <p slot="end"> {{ $filters.getShippingFrom(item.orderRoles, '1/Ship-From Vendor/') ? $filters.getShippingFrom(item.orderRoles, '1/Ship-From Vendor/') : "-" }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("Location inventory") }} </ion-label>
+                      <p slot="end"> {{item.uniqueOrderItemsCount}} </p>
```

Comment:
Need to get inventory for given item at the location it is shipping from. Will require getting productID and checking inventory at the facilityID of the order item. @adityasharma7 what do you think?

Reply:
Sir, added logic to get the product inventory at a specific facility.